src/codemodder/llm.py

```diff
@@ -81,5 +91,27 @@ def setup_llm_client() -> OpenAI | None:
     return OpenAI(api_key=api_key)
 
 
+def setup_azure_llama_llm_client() -> ChatCompletionsClient | None:
+    """Configure the Azure Llama LLM client."""
+    if not ChatCompletionsClient:
+        logger.info("Azure API client not available")
+        return None
+
+    azure_llama_key = os.getenv("CODEMODDER_AZURE_LLAMA_API_KEY")
+    azure_llama_endpoint = os.getenv("CODEMODDER_AZURE_LLAMA_ENDPOINT")
+    if bool(azure_llama_key) ^ bool(azure_llama_endpoint):
```

Comment:
This is my fault because I did this originally but `!=` is definitely a lot clearer here 😅

Reply:
yeah I kinda agree but I also got very used to reading this and it's nice :)